ROMUtils.cpp

```diff
@@ -55,11 +55,75 @@ namespace ROMUtils
     /// A pointer into the ROM data to start reading from.
     /// </param>
     /// <param name="outputSize">
-    /// The predicted size of the output data.
+    /// The predicted size of the output data.(unit: Byte)
     /// </param>
     /// <return>A pointer to decompressed data.</return>
-    unsigned char *RLEDecompress(unsigned char *data, int outputSize)
+    unsigned char *RLEDecompress(unsigned char *data, int address, int outputSize)
     {
-        // TODO
+        unsigned char *OutputLayerData = new unsigned char[outputSize];
```

Comment:
We should probably mention in the `<remarks>` or `<return>` documentation that the returned array is on the heap, and must be freed with the `delete` keyword when you are done using the array

Reply:
We also should probably call this variable something else, like DecompressedData because maybe there are other things in the ROM that use RLE compression.

Reply:
LGTM